package/src/prmon.cpp

```diff
@@ -2,12 +2,15 @@
   Copyright (C) 2018, CERN
 */
 
+#include <algorithm>
```

Comment:
I don't think we need this...

Reply:
Good catch, I was playing around and this a left over from that :) I took that out.

Reply:
Ha, yes, it's easy on the diff, but we should probably run IWYU properly too